### ISSUE TITLE
Create linux/arm64 images

### ISSUE THREAD

**Issue by sparty02** [gvenzl/oci-oracle-xe#44]

**Comment by gvenzl** [comment]
Hey @sparty02,

Oracle Database is not supported for ARM (see [this GitHub discussion for more info](https://github.com/oracle/docker-images/discussions/1951)).

That being said, I can see the value of adding the `linux/amd64` platform tag to the images, thanks!

**Comment by metawave** [comment]
There is some light after all: https://blogs.oracle.com/database/post/announcing-oracle-database-19c-support-for-arm

Unfortunately, they don't say anything about XE versions, fingers crossed they add ARM support in all the db products.

Using Colima x86 on M1Pro works, but then all your containers will use x86, some fail miserably. Still a hassle to startup multiple colima instances (x86 and ARM) for a simple docker-compose to run.....

**Comment by metawave** [comment]
@kmandalas I tested it again with Docker Desktop 4.25.something no luck and I highly doubt it will ever work on Docker Desktop. It's not a rosetta problem! It lies somewhere in the compatibility layer in arm64 operating systems when supporting x86 programs. To ensure it's no rosetta problem I also tested it on an ampere cpu server (also arm64 cpu) system with debian linux, the same problem. But I can confirm it still works using colima or lima docker when using an x86 os.

**Comment by uesleilima** [comment]
yeah, it is officially released a version for arm64 platform for 19c and the image is available at https://container-registry.oracle.com/ords/ocr/ba/database/enterprise under the tag 19.19.0.0, fingers crossed for a XE  or even a slim version would come shortly

**Comment by metawave** [comment]
arm64 support arrived. Thanks to @gvenzl. It's available in this projects successor https://github.com/gvenzl/oci-oracle-free. If you can use Oracle 23, there are now multi-arch images available. I don't think it's realistic to expect back-port to older Oracle versions. I'll plan to upgrade my projects to oci-oracle-free in a few days/weeks.

**Comment by gvenzl** [comment]
Thanks a lot, @metawave!

Indeed, Oracle is not going to backport ARM support for Oracle XE 21c or Oracle XE 18c so for folks who require/want Oracle Database on their ARM-based machines, aka Mac Books, please upgrade to `gvenzl/oracle-free`
